geode-core/src/main/java/org/apache/geode/cache/client/internal/QueueManagerImpl.java

```diff
@@ -309,6 +316,12 @@ public void serverRemoved(ServerLocation location) {
   }
 
 
+  @TestOnly
```

Comment:
This annotation alone has little visibility when writing code. I suggest the method name warn developers away from it. If used for testing it should probably be package-private and move the test into the same package.

Reply:
Done